docs/how-to/manage-users.md

```diff
@@ -0,0 +1,78 @@
+In this document, a user refers to a developer using fleet, not an end user accessing the magento site. Users are primarily used to manage authentication and authorization.
+
+Listing existing users
+----
+
+```
+$ fleet auth user list
+UUID                                 Email            Roles
+-----------------------------------  ---------------  -------
+4bb1538b-30f1-4017-9d82-6a5b3a407534 test@example.com Admin
+```
+
+The UUID is a unique identity generated for each user that never changes. However most actions accept an email as that is more readable.
+
+Creating a user
+----
+
+You can add [roles](/how-to/manage-roles) to a user when you create them, or you can do that separately. A user with no roles will not be authorized to do anything and has no authentication certificates.
+
+```
+$ fleet auth user add test@example.com
+Added user: test@example.com
+```
+
+When adding roles use a comma separated list:
+
+```
+$ fleet auth user add test@example.com --roles Developer,Accountant
+$ fleet auth user add test@example.com --roles Developer
+$ fleet auth user add test@example.com --roles "Developer, CertManager"
+```
+
+When you create a new user a verification email will be sent to their email address. The user needs to verify their account before they can do anything.
+
+Verifying a user
+----
+
+Verifying a user will also create a [certificate](/how-to/manage-certs) for them. Thus verification requires an email address, a label for the certificate, and a [private key file](/how-to/manage-certs#creating-an-auth-cert).
```

Comment:
A user must verify themself using the code sent via the email sent when their user was created.

Make it clear that you can't verify someone else.